### ISSUE TITLE
Strange behavior for Spyder code annotations

### ISSUE THREAD

**Issue by apdcode** [spyder-ide/spyder#6520]
## Problem Description
Using comments such as #TIPS and #HINTS will render a blue checkmark in the Spyder IDE. If you go to <kbd>Tools|Editor|Code Introspection/Analysis</kbd> and activate <kbd>Code Annotations (TODO, FIXME, XXX, HINT, TIP...)</kbd>, those keywords do not seem to trigger any effect in the SPyder IDE at all. 

### What steps will reproduce the problem?

1.  Go to Tools|Editor|Code Introspection/Analysis
2. Activate Code Annotations (TODO, FIXME, XXX, HINT, TIP...)
3. No blue checkmarks activated.

### What is the expected output? What do you see instead?
Covered in the Problem Description.

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Package Versions

* Spyder: 3.2.5
* Python: 3.6.0
* Qt:
* PyQt:
* Operating System: Windows

### Dependencies

```
PASTE DEPENDENCIES HERE
```

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll take a loot at this for 3.2.8

**Comment by csabella** [comment]
Can you give more of an example of what you are entering in order to get the blue checkmark?  For example, I noticed that 
```
# XXX
```
does not give a checkmark, but
```
# XXX - come back later
```
does.  I think the code checks that the comment has more text on the line than just the TODO, FIXME, etc.

**Comment by apdcode** [comment]
Hi!

I'd be happy to give more examples!

But regarding today's notification Closed #6520, are you still interested?

Anyway, thank you for taking the time to look at something that must only
be a minor detail in an otherwise great IDE!

**Comment by csabella** [comment]
@apdcode   It was closed because I submitted a patch for it.  The only time I was not getting the checkmarks was when the line had a `# TODO` or other todo-type word that wasn't followed by any other text.  So, I made a change to look for missing text and default it to the TODO word.
